packages/ipfs-core/src/components/files/utils/hamt-utils.js

```diff
@@ -102,6 +117,7 @@ const generatePath = async (context, fileName, rootNode) => {
       prefix: toPrefix(currentBucket._posAtParent)
     })
 
+    // @ts-ignore
```

Comment:
can we get a reason in the comment please ?

Reply:
Done